css/webstyle.css

```diff
@@ -0,0 +1,346 @@
+body {
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  height: 7.86vh;
+  background-color: #fff;
+  align-items: baseline;
+}
+
+h3 {
+  flex: 1;
+  margin-left: 10%;
+  color: #6070ff;
+}
+
+nav {
+  flex: 2;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 10%;
+}
+
+.hide {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  padding-inline-start: 0;
+  align-self: stretch;
+}
+
+ul :first-child {
+  padding-inline-start: 0;
+}
+
+li {
+  list-style: none;
+  padding-left: 12px;
+  padding-right: 12px;
+  margin-right: 8px;
+  font-weight: 500;
+}
+
+#hamb {
+  display: none;
+}
+
+main {
+  background-color: #fff;
+  background-image: url(../images/Headerbg.png);
+  background-size: 96%;
+  background-repeat: no-repeat;
+  background-position: 0 31%;
+  border-bottom-left-radius: 8.5% 8.5%;
+}
+
+@media (min-width: 1440px) {
+  main {
+    background-size: 93%;
+    background-position: right bottom;
+  }
+}
+
+@media (max-width: 375px) {
+  main {
+    background-image: url(../images/header-shapes.png);
+    background-size: contain;
+    border-bottom-left-radius: 70px 70px;
+  }
+}
+
+#main-section {
+  margin: 17%;
+}
+
+h1 {
+  color: #091e42;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 60px;
+  letter-spacing: 0.37px;
+  margin-block-end: 0;
+}
+
+#main-text {
+  margin-block-start: 12px;
+  color: #344563;
+  padding-right: 2%;
+  margin-block-end: 12px;
+}
+
+p {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+h4 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  color: #7f8cff;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: 80.0277%;
+  align-self: center;
+  background-color: #fff;
+  margin: 9.97%;
+  margin-bottom: 0;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
```

Comment:
- [ ] The card section is too big + it's not displaying the images.


- **Your card**

![your_card](https://user-images.githubusercontent.com/61562626/134772145-939455db-315f-4026-a1fc-e8e4d6e4c203.jpeg)


- **Figma template**

![cards](https://user-images.githubusercontent.com/61562626/134772160-72a70414-0c08-461f-b014-ecd1ca3653eb.jpeg)

Reply:
screenshot from index.html